org.eclipse.winery.repository.ui/src/app/section/section.component.ts

```diff
@@ -163,10 +165,29 @@ export class SectionComponent implements OnInit, OnDestroy {
 
         this.fileUploader.getUploader().setOptions({
             url: this.fileUploadUrl,
-            additionalParameter: {'overwrite': this.overwriteValue}
+            additionalParameter: { 'overwrite': this.overwriteValue }
         });
     }
 
+    applyNamespace() {
+        localStorage.setItem('initNamespace', this.initNamespaceString);
+        this.newComponentNamespace = this.initNamespaceString !== '' ? this.applyToscaTypeToNamespace(this.initNamespaceString) : '';
+    }
+
+    onAddModalShown() {
+        this.initNamespaceString = localStorage.getItem('initNamespace') !== '' ? localStorage.getItem('initNamespace') : 'http://www.example.org/tosca';
```

Comment:
Please use an enum for the localstorage entry and the default namespace like this:
```
enum StartNamespace {
    localStorageEntry = 'defaultNamespace',
    defaultStartNamespace = 'http://www.example.org/tosca'
}
```

This eases refactoring and reading.